libs/openFrameworks/3d/ofNode.h

```diff
@@ -1,10 +1,10 @@
 
 #pragma once
 
-#include "ofConstants.h"
 #include "ofParameter.h"
 #include <array>
 #include "glm/mat4x4.hpp"
+#include "ofConstants.h"
```

Comment:
might be good to have this above the glm line just incase it catches any non initialized errors